src/cluster_migrateslots.c

```diff
@@ -2604,6 +2611,11 @@ void clusterCommandSyncSlots(client *c) {
         clusterCommandSyncSlotsAck(c);
         return;
     }
+    if (!strcasecmp(c->argv[2]->ptr, "capa")) {
+        /* CLUSTER SYNCSLOTS CAPA <field> [CAPA <field>...] */
```

Comment:
The word CAPA shouldn't be repeated. (It isn't for REPLCONF CAPA.)

```suggestion
        /* CLUSTER SYNCSLOTS CAPA <field> [<field>...] */
```